### ISSUE TITLE
Update to Presentation plugin broken?

### ISSUE THREAD

**Issue by cmadland** [OleVik/grav-plugin-presentation#14]
I just updated the presentation deckset plugin from 1.3.1 to 1.3.3 and get the following error

![image](https://user-images.githubusercontent.com/12282929/71289402-ddc83580-2321-11ea-87bb-925da14a71e8.png)

Another repo looks like this...

![image](https://user-images.githubusercontent.com/12282929/71289487-1c5df000-2322-11ea-83c3-6d7f38a04a46.png)

There are no navigation arrows in the presentations either.

**Comment by OleVik** [comment]
That's odd, the change only prevents the DecksetParser from loading if the parent class is unavailable. Do you have a test-installation available that I could debug?

**Comment by cmadland** [comment]
I've invited you to a repo at https://github.com/TWUOnline/arts-master where the plugins are gitsynced.

**Comment by OleVik** [comment]
How did you install the Presentation-plugin? It was missing it's JavaScript-dependencies, which by default are included in most ways of installing it. The initial error is somewhat intermittent between tests, and I haven't been able to rectify it yet. The later error is visible in the browser's console, and it's actually the Zooming-plugin that breaks the Presentation when it's JS-assets are back in place.

**Comment by OleVik** [comment]
There's a chance that some local Git-config might have globally ignored /node_modules, even though the plugin unignores them itself:

```
node_modules/
!node_modules/reveal.js/css/reveal.css
!node_modules/reveal.js/css/theme/**/*.css
!node_modules/reveal.js/css/print/**/*.css
!node_modules/reveal.js/js/reveal.js
!node_modules/reveal.js/lib/js/head.min.js
!node_modules/reveal.js/lib/font/**/*
!node_modules/axios/dist/**/*.js
!node_modules/js-base64/base64.min.js
!node_modules/codemirror/lib/codemirror.js
```

**Comment by OleVik** [comment]
The initial error should be solved by [v1.3.4](https://github.com/OleVik/grav-plugin-presentation-deckset/releases/tag/v1.3.4) of DecksetParser.
